CHANGELOG.md

```diff
@@ -96,6 +125,10 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 - Loading Docker images as part of `WITH DOCKER` is now faster through the use of an embedded registry in Buildkit. This functionality was previously hidden (`VERSION --use-registry-for-with-docker`) and was only auto-enabled for Earthly Satellite users. It is now enabled by default for all builds. [#1268](https://github.com/earthly/earthly/issues/1268)
 
+### Changed
+
+- `VERSION` command is now required.
+
```

Comment:
I would move `### Changed` as the first section. This is the stuff that will contain breaking changes too and it feels pretty important.